R/plumber-options.R

```diff
@@ -1,60 +1,84 @@
-
-#' How to use Plumber options
+#' Plumber options
 #'
-#' @section Options:
 #' There are a number of global options that affect Plumber's behavior. These can
 #' be set globally with [options()] or with [options_plumber()]. Options set using
 #' [options_plumber()] should not include the `plumber.` prefix.
 #'
 #' \describe{
-#' \item{`plumber.apiHost` (defaults to `host` defined by `run` method, or an empty string
-#' when used outside a running router)}{Host used to build UI url and server url for
-#' OpenAPI specification.}
-#' \item{`plumber.debug` (defaults to `FALSE`)}{Provides more insight into your API errors. Alternatively,
-#' use parameter `debug` of plumber router `run` method}
-#' \item{`plumber.maxRequestSize` (defaults to `0`)}{Maximum length in bytes of request body. Body larger
-#' than maximum are rejected with http error 413. `0` means unlimited size.}
-#' \item{`plumber.postBody` (defaults to `TRUE`)}{Copy post body content to `req$postBody` using system encoding.
+#' \item{`plumber.apiScheme`}{Scheme used to build UI url and server url for
+#' OpenAPI Specification. Defaults to `http`, or an empty string
+#' when used outside a running router}
+#' \item{`plumber.apiHost`}{Host used to build UI url and server url for
+#' OpenAPI Specification. Defaults to `host` defined by `run` method, or an empty string
+#' when used outside a running router}
+#' \item{`plumber.apiPort`}{Port used to build UI url and server url for
+#' OpenAPI Specification. Defaults to `port` defined by `run` method, or an empty string
+#' when used outside a running router}
+#' \item{`plumber.apiPath`}{Path used to build UI url and server url for
+#' OpenAPI Specification. Defaults to an empty string}
+#' \item{`plumber.apiURL`}{UI url and server urls for OpenAPI Specification respecting
+#' pattern `scheme://host:port/path`. Defaults to the combination of the above options}
```

Comment:
Put `plumber.apiURL` first and make it more clear that if you specify this option, the rest of the other `api*` options are ignored